ue-task-management/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "UETaskManagement",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "node-fetch": "^2.6.0",
+    "@adobe/aio-sdk": "^3.0.0",
+    "@adobe/exc-app": "^0.2.21",
+    "@adobe/react-spectrum": "^3.4.0",
+    "@adobe/uix-guest": "0.9.1",
+    "@react-spectrum/list": "^3.0.0-rc.0",
+    "@spectrum-icons/workflow": "^3.2.0",
+    "core-js": "^3.6.4",
+    "node-html-parser": "^5.4.2-0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-error-boundary": "^1.2.5",
+    "react-router-dom": "^6.3.0",
+    "regenerator-runtime": "^0.13.5"
+  },
+  "devDependencies": {
+    "jest": "^27.2.4",
+    "@openwhisk/wskdebug": "^1.3.0",
+    "@babel/core": "^7.8.7",
+    "@babel/plugin-transform-react-jsx": "^7.8.3",
+    "@babel/polyfill": "^7.8.7",
+    "@babel/preset-env": "^7.8.7"
+  },
+  "engines": {
+    "node": "^14.18 || ^16.13 || >=18"
```

Comment:
Let's left only node support for 18th, the previous versions are EOL https://endoflife.date/nodejs